setup.cfg

```
@@ -0,0 +1,70 @@
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
+# Copyright (C) 2008-2018 NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+[metadata]
+name = cylc
+author = Hilary Oliver
+author_email = cylc@googlegroups.com
+url=https://cylc.github.io/
+description =
+    Cylc (\"silk\") is a workflow engine for cycling systems -
+    it orchestrates distributed suites of interdependent
+    cycling tasks that may continue to run indefinitely.
+keywords =
+    cycling-workflows
+    hpc
+    job-scheduler
+    metascheduler
+    workflow-automation
+    workflow-engine
+    workflow-management
+    scheduling
+license = GPL
+license_file = COPYING
+platforms = any
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Development Status :: 6 - Mature
+    Environment :: Console
+    Environment :: Web Environment
+    Intended Audience :: Developers
+    Intended Audience :: System Administrators
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: GNU General Public License v3 (GPLv3)
+    Operating System :: POSIX :: Linux
+    Programming Language :: Python
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: Implementation :: CPython
+    Topic :: Scientific/Engineering :: Atmospheric Science
+python_requires = >=3.5
```

I've used asyncio enhancements which pin us at `3.6+`:

```suggestion
python_requires = >=3.6
```

---

Updated, thanks for spot!